### PR TITLE
attempt to fix tests on windows

### PR DESCRIPTION
**Pull request by lifeart**
related: https://github.com/embroider-build/embroider/issues/676

cc @rwwagner90

### CODE REVIEW

File: packages/compat/src/audit.ts [hunk 1]
```diff
@@ -251,16 +251,34 @@ export class Audit {
...
+      // eslint-disable-next-line @typescript-eslint/no-require-imports
+      let templateCompiler = require(join(this.appDir, this.meta['template-compiler'].filename));
+      dbg.push('require');
+
+      let resolver = templateCompiler.params.resolver as CompatResolver;
+      dbg.push('resolver');
+
+
+      resolver.enableAuditMode();
+      dbg.push('enableAuditMode');
```
Comment (lifeart) [inline packages/compat/src/audit.ts:268]: trying to debugg exact error

**Comment by lifeart** [comment]
getting few strange test issues:

image


image

**Comment by lifeart** [comment]
one more thing to fix: 
image

**Comment by ef4** [comment]
Thanks. Once we land this we should definitely add a windows test to github actions so this doesn't regress again.

**Pull request closed without merging**